### PR TITLE
Updated

### PR DESCRIPTION
**Pull request by SheryarAli786**
Created different components:
Header
Modal
NoteComponent
styling with styled component

### CODE REVIEW

File: src/app/components/Modal/ModalStyle.tsx [hunk 1]
```diff
@@ -0,0 +1,126 @@
+import styled from "styled-components";
```
Comment (suleman-se) [inline src/app/components/Modal/ModalStyle.tsx:1]: it should be a ts file

File: src/app/components/Modal/ModalStyle.tsx [hunk 2]
```diff
@@ -0,0 +1,126 @@
...
+export const MainContainer=styled.div`
+position: absolute;
+right: 300px;
+z-index: 3;
```
Comment (suleman-se) [inline src/app/components/Modal/ModalStyle.tsx:5]: please align/format the text

File: src/app/components/Modal/ModalStyle.tsx [hunk 3]
```diff
@@ -0,0 +1,126 @@
...
+    height: 420px;
+    margin: auto;
+    width: 600px;
+    box-shadow: rgba(100, 100, 111, 0.2) 0px 7px 29px 0px;
+    border-radius: 10px;
+    background-color: white;
+`
+export const InsideContainer1=styled.div`
```
Comment (suleman-se) [inline src/app/components/Modal/ModalStyle.tsx:13]: give proper names

File: src/app/components/Notes/NotesComponent.tsx [hunk 4]
```diff
@@ -0,0 +1,43 @@
...
+  const isLocalStorageAvailable = typeof window !== 'undefined' && window.localStorage;
+
+
+  const savedData = isLocalStorageAvailable
+    ? JSON.parse(localStorage.getItem('myNotes') ?? '[]') 
+    : [];
+
+  return (
+    <Container>
+      {savedData.map((item: NoteItem, i: number) => (
```
Comment (suleman-se) [inline src/app/components/Notes/NotesComponent.tsx:23]: no need to add datatype to i

File: src/app/components/Notes/NotesComponentStyle.tsx [hunk 5]
```diff
@@ -0,0 +1,61 @@
...
+
+export const Container=styled.div`
+    display: flex;
+    flex-wrap: wrap;
+    justify-content: start;
+    position: relative;
+`
+export const MainContainer=styled.div`
+display: flex;
+flex-direction: column;
```
Comment (suleman-se) [inline src/app/components/Notes/NotesComponentStyle.tsx:11]: again same: format code

File: src/app/components/header/Header.tsx [hunk 6]
```diff
@@ -0,0 +1,31 @@
...
+const Header = () => {
+  const [isModelOpen, setIsModelOpen] = useState(false);
+  const openModel=()=>{
+    setIsModelOpen(true)
+  }
+  const closeModel=()=>{
+    setIsModelOpen(false)
+  }
+  return (
+    <div>
```
Comment (suleman-se) [inline src/app/components/header/Header.tsx:15]: remove div add modal to wrapper

File: src/app/components/header/HeaderStyleCom.tsx [hunk 7]
```diff
@@ -0,0 +1,63 @@
+// CommonBtnStyle.tsx
+import styled from "styled-components";
+export const WrapperContainer=styled.div`
+display:flex;
+padding-top:20px;
+align-items: center;
+
```
Comment (suleman-se) [inline src/app/components/header/HeaderStyleCom.tsx:7]: format code

File: src/app/components/Modal/ModalStyle.tsx (hunk 1 above)
Comment (SheryarAli786) [reply src/app/components/Modal/ModalStyle.tsx:1]: Sir Changes have been applied

**Pull request merged**